config/config.go

```diff
@@ -659,7 +659,7 @@ var defaultConf = Config{
 		TxnTotalSizeLimit:    DefTxnTotalSizeLimit,
 		DistinctAggPushDown:  false,
 		CommitterConcurrency: 16,
-		MaxTxnTTL:            10 * 60 * 1000, // 10min
+		MaxTxnTTL:            60 * 60 * 1000, // 1hour
```

Comment:
Do we use this value based on the feedback from the users?

Reply:
It is based on some feedbacks from our DBA team.